specs/engage-digital_openapi3.yaml

```diff
@@ -7591,6 +7782,21 @@ components:
           format: string
         utc_offset:
           format: integer
+    UserCapacity:
+      properties:

```

Comment:
`id`, `created_at` and `updated_at` are missing here

Reply:
Nice catch, and the props should be in alphabetical order